CONTRIBUTING.md

```diff
@@ -65,11 +76,16 @@ When a pull request is first created, @opendatakit-bot tags it as `needs review`
 
 Small fixes that target very particular bugs may occasionally be merged without a second review.
 
-## Triage Issues [![Open Source Helpers](https://www.codetriage.com/opendatakit/collect/badges/users.svg)](https://www.codetriage.com/opendatakit/collect)
+## Work in progress pull requests
+Work in progress (WIP) pull requests are useful to illustrate a proposed direction and get early feedback before committing to a solution direction. Work in progress pull requests should be clearly identified with [WIP] at the beginning of the title. The description should describe the proposed direction, why it is being considered, what alternatives might be possible and solicit specific feedback from collaborators. 
+
+A broad question like "what do you think of this?" makes it very difficult for a collaborator to engage with the pull request without coming up with a solution to the problem him or herself. Instead, please ask specific questions that make your own thinking clear and allow the reviewer to identify controversial parts of the proposed solution. For example: "here I returned a result object to represent the status after the transaction. I also considered throwing an exception in case of error but I didn't like that it made it unclear where an error happened. Which do you prefer and why?" 
```

Comment:
I think this example should be put in the `The review process` section to indicate what's a good question during review.

Reply:
That's very true!

Reply:
That's very true!

Reply:
Done.